src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {(isMarried) ? `${partnerName} is my ${partner}` : `I am not married`}
```

Comment:
parentheses is redundant here

```suggestion
        {isMarried ? `${partnerName} is my ${partner}` : `I am not married`}
```